config/stylelint-config-carbon/package.json

```diff
@@ -3,6 +3,7 @@
   "description": "Stylelint configuration for Carbon",
   "version": "1.19.0",
   "license": "Apache-2.0",
+  "type": "module",
```

Comment:
We'll want to cut a new major for this package once this PR is in. We can pair up and do it together if you'd like!